test/parser/samples/error-empty-classname-binding/input.svelte

```diff
@@ -1 +0,0 @@
-<h1 class:={true}>Hello</h1>
```

Comment:
We should keep this test, and update the error message in the error.json.

Reply:
I thought the same thing when writing the test. but I removed because they will throw the same error anyway:
```svelte
<h1 class:={true}>Hello</h1>
<h1 class:>Hello</h1>
```

Reply:
Done. I updated the error message on `"error-empty-classname-binding"` and kept the new test `"error-empty-directive-name"`

Reply:
ah that's great!

just want to make sure we complain for empty `class:` as well, not just empty `use:`